### PR TITLE
Turn Atoms into Images!

### LINKED ISSUE #332: Replace atoms with text images
Replace drawing atoms with text to drawing them with fixed size images.

This is the definitive solution for #138 and #278  to ensure atoms are the same size cross browser and after resize.

Images of letters should be placed in a subfolder of the the `/public/` directory.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by RyanR712** [comment]
Marking as ready for review. All letters load and change color according to theme

File: src/DrawTools/AtomTool.ts [hunk 1]
```diff
@@ -36,8 +35,8 @@ let wasOut: boolean;
 //True if the mouse button is currently down.
 let hasMouseDown: boolean;
 
-//AtomNode we are creating. Defaults to A at position (0, 0) on canvas.
-let currentAtom: AtomNode = createAtom("A", new Point(0, 0));
+//AtomNode we are creating.
+let currentAtom: AtomNode;
```
Comment (James-Oswald) [inline src/DrawTools/AtomTool.ts:39]: I liked how the old version gave us a default atom value
We should keep something like this in the new version.

File: src/AEG/AtomNode.ts [hunk 2]
```diff
@@ -157,7 +157,7 @@ export class AtomNode {
      */
     public calcRect(): Rectangle {
         return new Rectangle(
-            new Point(this.internalOrigin.x, this.internalOrigin.y - this.internalHeight),
+            new Point(this.internalOrigin.x, this.internalOrigin.y),
```
Comment (James-Oswald) [inline src/AEG/AtomNode.ts:160]: I personally preferred the mouse at the bottom of the atom

File: src/SharedToolUtils/DrawUtils.ts [hunk 3]
```diff
@@ -34,6 +52,132 @@ const atomCheckBox = <HTMLInputElement>document.getElementById("atomBox");
...
+    letterMap[42] = "q";
+    letterMap[43] = "r";
+    letterMap[44] = "s";
+    letterMap[45] = "t";
+    letterMap[46] = "u";
+    letterMap[47] = "v";
+    letterMap[48] = "w";
+    letterMap[49] = "x";
+    letterMap[50] = "y";
+    letterMap[51] = "z";
```
Comment (James-Oswald) [inline src/SharedToolUtils/DrawUtils.ts:112]: Do this with two for loops over the upper and lowercase alphabets respectively. 

Something like... 
```
for(let i = 0; i < 27; i++)
    letterMap.push(String.fromCharCode(97 + i));
for(let i = 0; i < 27; i++)
    letterMap.push(String.fromCharCode(65 + i)); 
```

**Review by James-Oswald — COMMENTED** [review]
Good work

File: src/AEG/AtomNode.ts (hunk 2 above)
Comment (RyanR712) [reply src/AEG/AtomNode.ts:160]: Determined not as important as keeping tests running on 07/26/2024 at 7:55 PM.

**Review by James-Oswald — APPROVED** [review]
Please merge once you have reviewed my change. You are free to release this as 1.3.0

**Pull request merged**